asab/application.py

```diff
@@ -584,3 +589,36 @@ def time(self):
 		Return UTC unix timestamp using a loop time (a fast way how to get a wall clock time).
 		'''
 		return self.BaseTime + self.Loop.time()
+
+
+	# Housekeeping
+
+	def set_housekeeping_time_from_config(self):
```

Comment:
`_set_housekeeping_time_from_config()` ... we don't want people to call this method